.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
+dist/
```

Comment:
Will this break production?

We need to commit dist until @LeoDiazL  and @imaustink 's changes are ready.